src/aind_metadata_mapper/bergamo/session.py

```diff
@@ -462,93 +463,99 @@ def _transform(self, extracted_source: RawImageInfo) -> Session:
-                                        "scanfields"
-                                    ]["repetitions"]
-                                ),
-                                spiral_duration=photostim_groups[0]["rois"][1][
-                                    "scanfields"
-                                ]["duration"],
-                                inter_spiral_interval=photostim_groups[0][
-                                    "rois"
-                                ][2]["scanfields"]["duration"],
-                            ),
-                            PhotoStimulationGroup(
-                                group_index=(
-                                    self.job_settings.photo_stim_groups[1][
-                                        "group_index"
-                                    ]
-                                ),
-                                number_of_neurons=int(
-                                    np.array(
+                                        ]["powers"]
+                                    ),
+                                    number_trials=(
+                                        self.job_settings.photo_stim_groups[0][
+                                            "number_trials"
+                                        ]
+                                    ),
+                                    number_spirals=int(
                                         photostim_groups[0]["rois"][1][
                                             "scanfields"
-                                        ]["slmPattern"]
-                                    ).shape[0]
-                                ),
-                                stimulation_laser_power=int(
-                                    photostim_groups[0]["rois"][1][
-                                        "scanfields"
-                                    ]["powers"]
-                                ),
-                                number_trials=(
-                                    self.job_settings.photo_stim_groups[1][
-                                        "number_trials"
-                                    ]
+                                        ]["repetitions"]
+                                    ),
+                                    spiral_duration=photostim_groups[0][
+                                        "rois"
+                                    ][1]["scanfields"]["duration"],
+                                    inter_spiral_interval=photostim_groups[0][
+                                        "rois"
+                                    ][2]["scanfields"]["duration"],
                                 ),
-                                number_spirals=int(
-                                    photostim_groups[0]["rois"][1][
-                                        "scanfields"
-                                    ]["repetitions"]
+                                PhotoStimulationGroup(
```

Comment:
@jtyoung84 This looks like a duplicate

Reply:
I think they're close, but there is a slight difference with the group index